src/include/units/quantity_point.h

```diff
@@ -63,6 +67,11 @@ class quantity_point {
     requires std::is_constructible_v<quantity_type, Q>
   constexpr explicit quantity_point(const Q& q) : q_{q} {}
 
+  template<QuantityPointLike QP>
+  constexpr explicit quantity_point(const QP& qp)
+    requires std::is_constructible_v<quantity_type, decltype(quantity_point_like_traits<QP>::relative(qp))>
```

Comment:
`std::constructible_from<T>`?

Reply:
I've been using type traits since `quantity` uses `units::is_same_v`.

Reply:
Yeah, truth. Do you think we should switch to concepts usage in all constraints wherever possible?
`explicit(...)` does not count.

Reply:
Definitely. From what I understand, they're user-observable via the subsumption mechanism. I wouldn't do it, but inheriting and attempting to overload a constructor with one that uses `units::is_same_v` could prove to be hard work.